data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "timid",
+        "evs": [0, 0, 0, 252, 4, 252],
+        "teratype": ["water", "fairy"],
+        "ability": "quarkdrive"
+      }
+    ]
+  },
+  "urshifu": {
+    "weight": 9,
+    "sets": [
+      {
+        "species": "urshifu",
+        "weight": 55,
+        "moves": [
+          ["swordsdance"],
+          ["wickedblow"],
+          ["closecombat", "drainpunch"],
+          ["suckerpunch"]
+        ],
+        "item": ["focussash"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["dark", "ghost"],
+        "ability": "unseenfist"
+      },
+      {
+        "species": "urshifu",
+        "weight": 25,
+        "moves": [
+          ["swordsdance"],
+          ["wickedblow"],
+          ["closecombat", "drainpunch"],
+          ["suckerpunch"]
+        ],
+        "item": ["blackglasses"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["dark", "poison", "ghost"],
+        "ability": "unseenfist"
+      },
+      {
+        "species": "urshifu",
+        "weight": 5,
+        "moves": [
+          ["counter"],
+          ["wickedblow"],
+          ["closecombat", "drainpunch"],
+          ["suckerpunch"]
+        ],
+        "item": ["focussash"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["dark", "ghost"],
+        "ability": "unseenfist"
+      },
+      {
+        "species": "urshifu",
+        "weight": 5,
+        "moves": [
+          ["wickedblow"],
+          ["closecombat", "drainpunch"],
+          ["suckerpunch"],
+          ["uturn"]
+        ],
+        "item": ["choiceband"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["dark", "poison", "ghost"],
+        "ability": "unseenfist"
+      },
+      {
+        "species": "urshifu",
+        "weight": 5,
+        "moves": [
+          ["wickedblow"],
+          ["closecombat", "drainpunch"],
+          ["suckerpunch"],
+          ["ironhead"]
+        ],
+        "item": ["choiceband"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["steel"],
+        "wantsTera": true,
+        "ability": "unseenfist"
+      },
+      {
+        "species": "urshifu",
+        "weight": 5,
+        "moves": [["wickedblow"], ["closecombat"], ["suckerpunch"], ["uturn"]],
+        "item": ["choicescarf"],
+        "nature": "jolly",
+        "evs": [0, 252, 4, 0, 0, 252],
+        "teratype": ["dark"],
+        "ability": "unseenfist"
+      }
+    ]
+  },
+  "tinglu": {
+    "weight": 9,
+    "sets": [
+      {
+        "species": "tinglu",
+        "weight": 65,
+        "moves": [
+          ["stealthrock"],
+          ["ruination"],
+          ["earthquake", "heavyslam"],
+          ["whirlwind"]
+        ],
+        "item": ["sitrusberry"],
+        "nature": "impish",
+        "evs": [244, 4, 116, 0, 124, 20],
```

Comment:
This 20 in speed isn't doing anything other than outspeeding other ting-lus since hippowdon doesn't exist here. Move at least most of it out to other stats. Same goes for the set below.